### PR TITLE
add more docs about migrating on npm

### PR DESCRIPTION
**Pull request by pup**
adds information regarding moving npm packages under our scope

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
 ```
 
-That's all - you don't need to explain yourself for wanting to hand over maintenance. We understand, that's why we're here. Of course, you can add a description of what your project is about or anything you'd like, really.
+That's all - you don't need to explain yourself for wanting to hand over
+maintenance. We understand, that's why we're here. Of course, you can add a
+description of what your project is about or anything you'd like, really.
+
+## Transferring
+
+After you've made your request, one of our owners is going to invite you into
```
Comment (igalic) [inline MIGRATING.md]: *admins

File: MIGRATING.md [hunk 2]
```diff
@@ -1,15 +1,53 @@
...
+Then, you can simply run `npm publish` to republish your package under our
+scope. Of course, you should also update your documentation to reflect this
+change.
+
+It is also advisable that you deprecate the old name of your package. You can
+do this by running the following command:
+
+```sh
+npm deprecate <your-old-package-name> "This package is now available at @fruitful/<your-package-name>"
+```
```
Comment (igalic) [inline MIGRATING.md]: if it was fully-qualified to begin with?…

we should ask npm folks if they have some kind of redirect process in place? or is `npm deprecate` that process?

Comment (pup) [inline MIGRATING.md]: i think(???) that deprecate is the default process? but yeah we should ask

**Pull request merged**